packages/private/edtr-io/src/plugins/equations/editor.tsx

```diff
@@ -411,40 +374,30 @@ function InlineMath(props: InlineMathProps) {
     <MathEditor
       readOnly={!focused}
       state={`${prefix}${state.value}${suffix}`}
-      config={{
-        i18n: {
-          placeholder: props.placeholder,
-        },
-      }}
+      config={{ i18n: { placeholder: props.placeholder } }}
       inline
       disableBlock
       visual={preferences.getKey(preferenceKey) === true}
       onEditorChange={(visual) => {
         preferences.setKey(preferenceKey, visual)
       }}
       onInlineChange={() => {}}
-      onChange={(value) => {
-        onChange(value)
-      }}
-      onMoveOutRight={() => {
-        onFocusNext()
-      }}
-      onMoveOutLeft={() => {
-        onFocusPrevious()
-      }}
+      onChange={onChange}
+      onMoveOutRight={onFocusNext}
+      onMoveOutLeft={onFocusPrevious}
     />
   )
 }
 
 type GridFocusState = {
   row: number
   column: number
-} | null
+}
 
 interface GridFocus {
-  focus: GridFocusState
-  isFocused: (payload: { row: number; column: number }) => boolean
-  setFocus: (focus: GridFocusState) => void
+  focus: GridFocusState | null
+  isFocused: (cell: GridFocusState) => boolean
+  setFocus: (cell: GridFocusState) => void
```

Comment:
This worked with `null` previously (to clear the focus). Don't we need anymore?

Reply:
In the current implementation: No. When I remember it well it wasn't also needed when I started to code on this issue